File: scala/algorithms/warmup/SolveMeFirst.scala
package algorithms.warmup

// In order to submit to hackerrank, rename it to "Solution"
object SolveMeFirst {
  def main(args: Array[String]) {
    println(input.take(2).map(_.toInt).sum)
  }

  private def input: Iterator[String] = {
    io.Source.stdin.getLines()
  }
}

File: php/algorithms/warmup/SolveMeFirst.php
<?php
function solve_me_first($a, $b) {
  return $a + $b;
}

$handle = fopen ("php://stdin","r");

$a = fgets($handle);
$b = fgets($handle);

$sum = solve_me_first((int) $_a, (int) $_b);

print($sum);

fclose($handle);
?>
